compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -432,6 +424,15 @@ void CircleOptimizer::optimize(loco::Graph *g) const
   {
     phase.emplace_back(std::make_unique<luci::UnrollUnidirectionalSequenceLSTMPass>());
   }
+  // Forward Reshape/Transpose after trying to remove redundant Reshape/Transpose
```

Comment:
As this PR #10596 explains detail in main comment, how about adding
`refer https://github.com/Samsung/ONE/pull/10596` comment too?

Reply:
I've updated the comment as you suggested.